app/models/user.rb

```diff
@@ -2,6 +2,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  has_many :account_invitations
+  has_many :my_accounts_invitations, class_name: 'AccountInvitation'
```

Comment:
```suggestion
  has_many :accounts_invitations
```
Put it back, let's not use my/his/etc. in the code. All things should have definite term. In the beginning, things can be named more vaguably, like here `account_invitations` is perfectly ok, it's clear what it means at this concrete time of the project. And when and only when you face naming issue, we change it.